measCompApp/Db/measCompBinaryOut.template

```diff
@@ -1,6 +1,7 @@
 record(bo, "$(P)$(R)")
 {
-    field(PINI, "$(PINI=YES)")
+#   field(PINI, "$(PINI=YES)")
```

Comment:
Adding DESC is not backwards compatible, you are forcing everyone to modify their files to include the DESC macro.

Reply:
OK, now DESC has an empty default value